tabs.js

```diff
@@ -0,0 +1,126 @@
+var h = require('hyperscript')
+var u = require('./lib/util')
+  each = u.each,
+  find = u.find,
+  isVisible = u.isVisible,
+  setVisible = u.setVisible,
+  setInvisible = u.setInvisible
+
+function toggle_focus(page) {
+  isVisible(page)
+    ? blur(page)
+    : focus(page)
+}
+
+function focus(page) {
+  if (isVisible(page)) return
+
+  setVisible(page)
+  var el = page.firstChild
+  el.dispatchEvent(new CustomEvent('focus', {target: el}))
+}
+
+function blur (page) {
+  if (!isVisible(page)) return
+
+  setInvisible(page)
+  var el = page.firstChild
+  el.dispatchEvent(new CustomEvent('blur', {target: el}))
+}
+
+function moveTo(page, content, i) {
+  if(!content.children.length || i >= content.children.length)
+    content.appendChild(page)
+  else
+    content.insertBefore(page, content.children[i])
+}
+
+module.exports = function (content, onSelect) {
+  var tabs = h('section', { className: '.tabs' })
+  var selection
+
+  function build_tab (page) {
+    var link = h('a', {
+      className: '.link',
```

Comment:
Hmmm, the options are `a.link` and `a.close`? 

I think these two things serve quite different roles. IMO using `.` makes more sense in this case because a close button is nothing like (even in styling) to the main tab link.

If there are no shared styles, then it should definitely be `.link`